src/khoj/routers/indexer.py

```diff
@@ -103,9 +85,9 @@ async def index_batch(
                 dict_to_update = plaintext_files
 
             if dict_to_update is not None:
-                dict_to_update[file.path] = file.content
+                dict_to_update[file.filename] = file.file.read().decode("utf-8")
```

Comment:
Does this work even if the file is binary encoded? Does it work for PDFs, and would it work for images as well?

Reply:
Umm, not sure but valid concern. Let me test that

Reply:
Fixed up and validated indexing of PDF files
  - 541cd59 Let fs_syncer pass PDF files directly as binary before indexing
  - d27dc71 Use encoding of each file set in indexer request to read file